### PR TITLE
Fix get_starred_packages

### PR DESCRIPTION
**Pull request by kokorinosoba**
Fixed bug due to change of output format of "apm starred"

### CODE REVIEW

**Review by amano41 — CHANGES_REQUESTED** [review]
Package names are printed in the third column only when the platform is darwin.
https://github.com/atom/apm/blob/master/src/stars.coffee#L80

Could you please add some codes to verify the platform?

**Comment by kokorinosoba** [comment]
Thanks for telling me. I didn't know that.
I changed the script to get the third column when using OSX.
But I couldn't check the operation with other os.
Could you please check it?

File: apm-sync.sh [hunk 1]
```diff
@@ -4,13 +4,34 @@ set -e
 set -u
 
 
+function is_darwin() {
+	if [ "$(uname | lower)" == 'darwin' ]; then
```
Comment (amano41) [inline apm-sync.sh]: I suppose 'lower' is your custom command.
Please use 'tr' for instance to ensure the portability.

```suggestion
	if [ "$(uname | tr '[:upper:]' '[:lower:]')" == 'darwin' ]; then
```

You can also just compare with a non-lowercased string (i.e., 'Darwin').

```suggestion
	if [ "$(uname)" == "Darwin" ]; then
```

File: apm-sync.sh [hunk 2]
```diff
@@ -4,13 +4,34 @@ set -e
...
+		exit 0
```
Comment (amano41) [inline apm-sync.sh]: The exit statement terminates the entire script with the specified exit code.
You should use the return statement in shell functions instead.

File: apm-sync.sh [hunk 3]
```diff
@@ -4,13 +4,34 @@ set -e
...
+	else
+		exit 1
```
Comment (amano41) [inline apm-sync.sh]: same as above

File: apm-sync.sh [hunk 4]
```diff
@@ -4,13 +4,34 @@ set -e
...
+	fi
+}
+
+
+function get_col() {
+
+	## Package names are printed in the third column only in darwin(osx)
+	## https://github.com/atom/apm/blob/master/src/stars.coffee#L80
+	if [ is_darwin ]; then
```
Comment (amano41) [inline apm-sync.sh]: No need to use '[ ]' when checking an exit status of a command/function.
In fact, the '[' (test command) in this case will always return true irrespective of the platform.

```suggestion
	if is_darwin; then
```

**Review by amano41 — CHANGES_REQUESTED** [review]
I tested your code and found some bugs.
Please check the comments below.
Thank you.

**Comment by amano41** [comment]
Thank you for your effort and cooperation.

There seem to be some bugs.
I'd appreciate if you could fix them.
Then could you squash commits if it is possible?

**Comment by kokorinosoba** [comment]
Thank you for reviewing my script!
Fixed everything you pointed out :-)

**Comment by amano41** [comment]
Thank you for your quick response.
Now this script runs correctly even on the mac. Yay!

**Pull request merged**